tools/scripts/server/stopServer.bat

```diff
@@ -19,13 +19,13 @@ echo Killing the (potentially) running server. This might take a while...
 	taskkill /im %exeName% > NUL 2>&1
 	
 	if [%errorlevel%] == [0] (
-		:: sleep 100ms
-		pathping localhost -n -q 1 -p 100 >nul
+		:: sleep 1s

```

Comment:
Waiting a full second in between every iteration is a pain in the *** and can increase the wait-time significantly. Why did you change a working solution?

Reply:
I removed pathping because it relies on specific firewall and DNS configuration which is a bit overkill to just wait a bit, and can fail catastsrophically when your setup differs. We can use windows' native sleep function "TIMEOUT". Unfortunately the smallest delay of it is 1 second, but that should not be a problem, because it only delays when an error occures - which never happened to me. Maybe my hard-kill modification in stopServer.bat helped to get rid of the file-lock.

Reply:
Ah I kinda missed that you changed the killing as well... Though I'm not quite sure if the arma server responds well to hard-killing... Iirc I read something about this causing problems if restarting the server immediately after :thinking:

Reply:
So I just tested it on my machine and the looping is absolutely required in order for the server to actually getting killed (when restarting it).
Thus waiting a full second in between the iterations isn't really acceptable...

Reply:
I think I found a solution: https://stackoverflow.com/a/29879492/3907364 (second option)
This uses JScript to do the waiting which is way more powerful than batch's timeout. Simply copy the provided code (and a link to that SO-question) in a batch-file called `sleep.bat` and then inside our batch code we write `call %~dp0.\sleep.bat 100` to wait 100 ms

Reply:
I watched that stopServer routine on my machine again as well. It takes 2-3 iterations (2-3 seconds) to get it (soft)-killed.
When we go back to your 100 ms delay, we would have 20 - 30 iterations and would statistically be able to restart the server 0.4 seconds earlier.
By looking at your SO-example i came to the conclusion that this 0.4 seconds are not worth it to include such a piece of garbage code :-) Then i'd rather go back to your traceping method or include some solid linux tools like "sleep" as .exe file. (We'll probably need more of them in the future anyway - sed, awk, cut, ...)

Reply:
Well using Linux tools is not really an option, is it? 'cause after all this has to run on windows... 
And I don't se what's wrong with the SO code. Noone has to look at it once it is there - and it does what it's asked for ^^

Reply:
ok i was looking at a different script from that page.
its integrated now - please merge this commits so i can delete the repo to fix my git problem.